tests/libvmi/vm.go

```diff
@@ -22,6 +22,7 @@ package libvmi
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
```

Comment:
At first view, I was not very happy about this additional dependency, but this dependency is real in the production code and in VM cr, so I don't see a way around having it.

Reply:
It is part of the API, no way around it.

Reply:
hence my lgtm ;-)